store/sqlstore/team_store.go

```diff
@@ -1194,6 +1198,7 @@ func (s SqlTeamStore) GetChannelUnreadsForAllTeams(excludeTeamId, userId string)
 	return data, nil
 }
 
+// GetChannelUnreadsForTeam returns all the unread messages in the channel for a single team
```

Comment:
Same problem here, we are returning the metadata (unreads/mentions/notifyProps) of all the channels in a team, but we are not returning the messages.

Reply:
fixed it